misc/snapshotter/config.toml

```diff
@@ -104,3 +104,19 @@ enable_stargz = false
 # The option enables trying to fetch the Nydus image associated with the OCI image and run it.
 # Also see https://github.com/opencontainers/distribution-spec/blob/main/spec.md#listing-referrers
 enable_referrer_detect = false
+[experimental.tarfs]
+# Whether to enable nydus tarfs mode
+enable_tarfs = false
```

Comment:
Should we remind users that they should use a specific version of nydus-image ?

Reply:
Changed the comment to
```
# Whether to enable nydus tarfs mode. The EROFS filesystem driver since Linux 6.4 supports tarfs mode.
```